### PR TITLE
PXB-3217 Add titles to the index page (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/backup-overview.md
modified:   docs/index.md
modified:   requirements.txt

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -43,7 +42,43 @@ Percona XtraDB Cluster.
...
+
+Find the best installation solution with our step-by-step installation instructions.
+
+[Installation instructions :material-arrow-right:](installation.md){ .md-button }
+
+</div><div data-banner markdown>
+
+## :fontawesome-solid-gears: Percona XtraBackup binaries { .title }
+
+Learn about Percona XtraBackup binaries.
```
Comment (patrickbirch) [inline docs/index.md]: Can we list the binaries? Learn about the Percona XtraBackup binaries: xtrabackup, xbcloud, xbcrypt and xbstream.

File: docs/index.md [hunk 2]
```diff
@@ -43,7 +42,43 @@ Percona XtraDB Cluster.
...
+
+[Percona XtraBackup binaries :material-arrow-right:](binaries-overview.md){.md-button}
+
+</div><div data-banner markdown>
+
+### :material-backup-restore: Backup management { .title }
+
+Learn about different types of backups and how to take them.
```
Comment (patrickbirch) [inline docs/index.md]: Learn about the different types

File: docs/index.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/index.md]: updated

File: docs/index.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/index.md]: updated

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**